src/test/groovy/life/qbic/utils/NanoporeParserSpec.groovy

```diff
@@ -78,23 +78,7 @@ class NanoporeParserSpec extends Specification {
         // Remove new created folder after testing
         directory.delete()
     }
-
-  def "parsing an empty directory throws ParseException"() {
-    given:
-    def pathToDirectory = Paths.get(exampleDirectoriesRoot, "/empty_directory/")
-    // Maven doesn't include empty folders in build process so it has to be generated explicitly
-    File directory = new File(pathToDirectory.toString())
-    if (!directory.exists()) {
-      directory.mkdir()
-    }
-    when:
-    NanoporeParser.parseFileStructure(pathToDirectory)
-    then:
-    thrown(ParseException)
-    // Remove new created folder after testing
-    directory.delete()
-  }
-
```

Comment:
why remove this test?

Reply:
The test is still in the current code, it was just duplicated from development during the merge conflict resolution. This removes just the duplication.

Reply:
Ok did not see that